### PR TITLE
SA-4 | Week-8 | Added send notification feature

### PR DESCRIPTION
**Pull request by vanshksharma**
## Issue that this pull request solves

Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [ ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [ ] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Review by rishiktejatanugula — COMMENTED** [review]
approve @akshatnema

**Comment by manavdesai27** [comment]
@vanshksharma Add module name and week number in PR name.

**Comment by vanshksharma** [comment]
@manavdesai27 Done!!

File: .idea/Fusion.iml [hunk 1]
```diff
@@ -17,7 +17,7 @@
       <sourceFolder url="file://$MODULE_DIR$/FusionIIIT" isTestSource="false" />
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.8 (Fusion)" jdkType="Python SDK" />
+    <orderEntry type="inheritedJdk" />
```
Comment (akshatnema) [inline .idea/Fusion.iml]: why did you make these changes?

Comment (vanshksharma) [reply .idea/Fusion.iml]: @akshatnema merge only those changes which are in the placement module section. Ignore the rest.

Comment (akshatnema) [reply .idea/Fusion.iml]: So, please remove those changes from this PR. There is no specific option to merge manual changes from the PR.

**Comment by vanshksharma** [comment]
@akshatnema  changes in files such as docker-entrypoint.sh,  script.go are not made by me but these are the changes incorporated by taking pull from the branch so merge them

**Comment by akshatnema** [comment]
> @akshatnema changes in files such as docker-entrypoint.sh, script.go are not made by me but these are the changes incorporated by taking pull from the branch so merge them

Pull the latest changes of remote sa-4 branch to your PR and then we will have correct files changed.

**Comment by vanshksharma** [comment]
@akshatnema All the changes are done. Now you can merge the PR!!

**Comment by akshatnema** [comment]
Changes in the modes.py and migrations added. Review required from @PranshuNayak and @samay-rgb.

@vanshksharma Kindly resolve the conflicts.

**Comment by vanshksharma** [comment]
@akshatnema  Done!!

**Review by PranshuNayak — COMMENTED** [review]
There are no database changes.The migrations contain renaming a field and reverting it to its previous name. Remove the migrations

**Comment by vanshksharma** [comment]
@akshatnema Why are you not  merging this PR??????????

**Comment by akshatnema** [comment]
@vanshksharma Pranshu asked you to remove the migrations. In the files changed, there are files related to migrations. Kindly, remove the changes of that file.

File: FusionIIIT/Fusion/settings/development.py
Comment (samay-rgb) [inline FusionIIIT/Fusion/settings/development.py:1]: Revert changes of this file.

**Comment by vanshksharma** [comment]
@akshatnema merge this PR. I have made changes to development.py

**Comment by akshatnema** [comment]
Review Approval of both @samay-rgb and @PranshuNayak are needed in this PR as new commits are being added.

**Comment by vanshksharma** [comment]
@samay-rgb  @PranshuNayak Kindly do the needful so that the PR can be merged at the earliest

File: FusionIIIT/applications/academic_information/models.py
Comment (samay-rgb) [inline FusionIIIT/applications/academic_information/models.py:1]: Why have you made these changes? These were added by another team.

File: FusionIIIT/applications/central_mess/admin.py
Comment (samay-rgb) [inline FusionIIIT/applications/central_mess/admin.py:1]: AIMB

**Pull request closed without merging**